loginCompose/src/main/res/drawable/ic_baseline_arrow_back_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:autoMirrored="true" android:height="24dp"
```

Comment:
Stuff like this may want to go to a UI core

Reply:
I agree, I have moved it to the ui core